src/Solana/Address.h

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../PublicKey.h"
+
+#include <array>
+#include <cstdint>
+#include <vector>
+
+namespace TW::Solana {
+
+class Address {
```

Comment:
Can we reuse `Base58Address.h` ? looks like that we can just override `Base58Address(const PublicKey& publicKey, const Data& prefix)`